connections/etuovi/services.py

```diff
@@ -19,6 +19,7 @@ def fetch_apartments_for_sale() -> list:
         Search()
         .query("match", _language="fi")
         .query("match", apartment_state_of_sale=ApartmentStateOfSale.FOR_SALE)
+        .query("match", publish_on_etuovi=True)
```

Comment:
I'm not so familiar with the API, but would it be possible to combine the queries to something like this:
```python
Search().query(
    "match",
    _language="fi",
    apartment_state_of_sale=ApartmentStateOfSale.FOR_SALE,
    publish_on_etuovi=True,
)
```
If so, then it would greatly simplify other parts of the code as well, for example `get_elastic_apartments_for_sale_uuids` and `publish_elastic_apartments`.

Reply:
that is unfortunately not possible.